src/main/java/com/utils/BasicTest.java

```diff
@@ -3,36 +3,39 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
-import io.github.bonigarcia.wdm.WebDriverManager;
-import java.util.concurrent.TimeUnit;
+
+import java.time.Duration;
 
 
 public abstract class BasicTest {
-    
-    public static final Logger logger = LogManager.getLogger();
+
     protected static WebDriver driver;
+
     // private String driverPath;
 
     @BeforeMethod
     public void preCondition() {
-        // Chromedriver path
-        // driverPath = "src/main/resources/WebDrivers/chromedriver.exe";
-        // ChromeOptions options = new ChromeOptions();
-        // System.setProperty("webdriver.chrome.driver", driverPath);
-        // driver = new ChromeDriver(options);
-        WebDriverManager.chromedriver().setup();
-        driver = new ChromeDriver();
+        //  Chromedriver path
+
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--headless");
```

Comment:
Cho nay neu em de nhu vay thi khi ma em run local no cung headless roi nhi. A suggest em if else khuc nay lay bien' tu environmenrt ve. Ex:

```
if("TRUE".equalsIgnoreCase(System.getenv("CHROME_HEADLESS"))){
                    options.addArguments("--headless");
                    LOGGER.info("Running automation with headless");
                }
```

Sau do tren file yaml e co the set env: CHROME_HEADLESS la true la ngon nha